R-package/cran-comments.md

```diff
@@ -1,5 +1,56 @@
 # CRAN Submission History
 
+## v3.0.0 - Submission 2 - (August 28, 2020)
+
+## v3.0.0 - Submission 1 - (August 24, 2020)
+
+NOTE: 3.0.0-1 was never released to CRAN. CRAN was on vacation August 14-24, 2020, and in that time version 3.0.0-1 (a release candidate) because 3.0.0.
```

Comment:
I'm sorry I can't get the meaning of the second sentence, it looks incomplete (around `because`)

> CRAN was on vacation August 14-24, 2020, and in that time version 3.0.0-1 (a release candidate) because 3.0.0.

Reply:
hah! it was supposed to be `became` not `because`

Reply:
```suggestion
NOTE: 3.0.0-1 was never released to CRAN. CRAN was on vacation August 14-24, 2020, and in that time version 3.0.0-1 (a release candidate) became 3.0.0.
```

Reply:
OK, now everything is clear!